lib/consultant.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
```

Comment:
since we have no bundler in this case, we have to require files in the order in which they are needed. This was a little tricky, but manageable. We'll need to make sure we continue this workflow as we add files